plugins/modules/zos_stat.py

```diff
@@ -2246,6 +2287,49 @@ def fill_return_json(attrs):
     return attrs
 
 
+def get_name_if_data_set_is_alias(name, module, tmp_hlq=None):
+    """Checks the catalog to see if 'name' corresponds to a data set
+    alias and returns the original data set name in case it is.
+
+    Arguments
+    ---------
+        name (str) -- Name of a data set or alias.
+        module (AnsibleModule) -- Ansible object with the task's context.
+        tmp_hlq (str, optional) -- Temp HLQ to use with mvscmdauth.
+
+    Returns
+    -------
+        bool -- Whether name corresponds to a data set alias.
+        str -- Name of the data set that the alias points to
+    """
+    # We need to unescape because this calls to system can handle
+    # special characters just fine.
+    name = name.upper().replace("\\", '')
+
+    stdin = f" LISTCAT ENTRIES('{name}') ALL"
+    cmd = "mvscmdauth --pgm=idcams --sysprint=* --sysin=stdin"
```

Comment:
We should start avoiding the use of zoau utilities in shell and prefer the python APIs when available, in this case please use https://www.ibm.com/docs/en/zoau/1.3.0?topic=apis-mvscmd